css/homestyle.css

```diff
@@ -0,0 +1,123 @@
+body, html {
+    padding: 0;
+    margin: 0;
+    height: 100%;
+}
+.coverImage {
```

Comment:
numele clasei nu e ok, dar o sa iti las comentariu in html ca sa intelegi despre ce e vorba